client/my-sites/plans/jetpack-plans/product-lightbox/style.scss

```diff
@@ -250,3 +256,10 @@
 	white-space: nowrap !important;
 	border: 0 !important;
 }
+
+// The lightbox z-index is 100200, consequences of updating the popover z-index
+// to higher than that are unclear, so we'll manually set it to 100200 here so we
+// can see the popover in the lightbox.
+.popover {
```

Comment:
Should this be more specific (with a class name added to the popover) to prevent styles seeping elsewhere after this stylesheet is loaded? Perhaps also included alongside the other z-index mappings: https://github.com/Automattic/wp-calypso/blob/trunk/client/assets/stylesheets/shared/functions/_z-index.scss

Reply:
I agree with that! Especially since `<InfoPopover />` allows for adding custom modifier class names, we should leverage that - instead of providing a global change as it's not our intention.

Reply:
Thank you both on this, I went ahead and added the class that was added to the component to specify it more. I can't nest it because the popover itself is rendered outside the parent component but this should keep it specific to this popover